<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>361</width>
      <height>381</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QTextBrowser" name="textBrowser"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Startwert Raumtemperatur [Â°C]:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btn_create_room">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>390</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Neuen Raum anlegen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_exec_sim">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Simulation starten</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_show_sim_data">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>430</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Raumdaten anzeigen</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>379</x>
      <y>-1</y>
      <width>132</width>
      <height>381</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lbl_room_nr">
       <property name="text">
        <string>Raumnr.:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_room_nm">
       <property name="text">
        <string>Raumbezeichnung:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_temp_sh">
       <property name="text">
        <string>Temperatur (SOLL):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_hum_sh">
       <property name="text">
        <string>Luftfeuchtigkeit (SOLL):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_thermostat_id">
       <property name="text">
        <string>Thermometer-ID:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_hygrometer_id">
       <property name="text">
        <string>Hygrometer-ID:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_rel_hum_is">
       <property name="text">
        <string>rel. Luftfeuchtigkeit (IST):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_abs_hum_is">
       <property name="text">
        <string>abs. Luftfeuchtigkeit (IST):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_time1">
       <property name="text">
        <string>Uhrzeit:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>0</y>
      <width>111</width>
      <height>381</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="lbl_rm_nr">
       <property name="text">
        <string>rm_nr</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_rm_nm">
       <property name="text">
        <string>rm_nm</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_tm_sh">
       <property name="text">
        <string>tm_sh</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_hm_sh">
       <property name="text">
        <string>hm_sh</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_tm_id">
       <property name="text">
        <string>tm_id</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_hy_id">
       <property name="text">
        <string>hy_id</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_rel_hm">
       <property name="text">
        <string>rel_hm</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_abs_hm">
       <property name="text">
        <string>abs_hm</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_time2">
       <property name="text">
        <string>time</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>629</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
